lib/src/wrappers/ResponsiveWrapper.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+import { useMediaQuery, Theme } from '@material-ui/core';
+import { ModalWrapperProps, ModalWrapper } from './ModalWrapper';
+import { InlineWrapperProps, InlineWrapper } from './InlineWrapper';
+import { Breakpoint } from '@material-ui/core/styles/createBreakpoints';
+
+export interface ResponsiveWrapperProps extends InlineWrapperProps, ModalWrapperProps {
+  /** Breakpoint where mobile picker will be changed to desktop
+   * @default 'sm'
+   */
+  desktopModeBreakpoint?: Breakpoint;
+}
+
+export const ResponsiveWrapper: React.FC<ResponsiveWrapperProps> = ({
+  desktopModeBreakpoint = 'sm',
+  okLabel,
+  cancelLabel,
+  clearLabel,
+  todayLabel,
+  showTodayButton,
+  clearable,
+  DialogProps,
+  PopoverProps,
+  ...other
+}) => {
+  const isDesktop = useMediaQuery<Theme>(theme => theme.breakpoints.up(desktopModeBreakpoint));
```

Comment:
We are lucky date pickers, on the server, aren't relevant :). Maybe we could add a logic to render null on the server, to clear up any potential confusion (I'm wondering) and save CPU cycles (I'm not too sure about this one because if it then render a Modal, it will be empty)?